### ISSUE TITLE
TS interface is incorrect for onIdle, onActive, onAction

### ISSUE THREAD

**Issue by Tolgeros** [SupremeTechnopriest/react-idle-timer#75]
From the interface, index.d.ts:
```
    /**
     * Function to call when user is idle
     */
    onIdle?: () => void;

    /**
     * Function to call when user becomes active
     */
    onActive?: () => void;

    /**
     * Function to call when user have an activity
     */
    onAction?: () => void;
```

Each of these prop functions are missing the Event argument, e.

**Comment by SupremeTechnopriest** [comment]
Ah, correct you are.  Easy fix.  Ill push a new version today.

**Comment by SupremeTechnopriest** [comment]
Hey @Tolgeros, published a fix for this as version `4.2.4`.  Let me know if there are any more issues!
